src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -13,20 +13,20 @@
   public static final String AD_DRIVER_FORCED_STATUS_SUCCESS_SCRIPT =
       "top.window.adDriver2ForcedStatus['%s'.split(',')[0]]='success';";
   //SLOTS NAMES
-  public static final String BOTTOM_LB = "BOTTOM_LEADERBOARD";
+  public static final String BOTTOM_LB = "bottom_leaderboard";
   public static final String MOBILE_BOTTOM_LB = "bottom_leaderboard";
-  public static final String TOP_LB = "TOP_LEADERBOARD";
+  public static final String TOP_LB = "top_leaderboard";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "top_leaderboard" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)